### DISCUSSION TITLE
[Delight Con Event] 여러분들이 좋아하는 앱을 소개해주세요

### DISCUSSION THREAD

**Discussion by yeniful** [DeveloperAcademy-POSTECH/.github#41] [General]
앱 이름 / 어떤 앱인지 / 왜 좋아하는지를 알려주세요!
사진도 함께 첨부해주시면 더 좋아요 🤗

**Comment by jihee-daily** [comment]
![image](https://github.com/user-attachments/assets/d27e74af-bf57-4b1d-a486-95036c6bcc2b)
![image](https://github.com/user-attachments/assets/02178433-b5f2-4fa5-95d3-e477c623ba34)
DailyArt. 이번 챌린지를 위해 벤치마킹하다가 알게된 앱이에요. 하루하루 추천 미술작품을 볼 수 있어서 좋습니다.

**Reply by yeniful** [reply]
우와! 매일 작품을 큐레이션을 해주는 앱이군요 +_+ 저도 한번 다운받아봐야겠어요 !! 글 서두에 기수와 닉네임도 함께 알려주시면 감사드리겠습니다-! ㅎㅎ

**Comment by Aengzii** [comment]
![image](https://github.com/user-attachments/assets/5f9f6cd4-612b-480f-b7d0-7833e99a4c0e)![image](https://github.com/user-attachments/assets/9f3c009e-be6c-4520-9e32-8b750566564f)

파워 대문자 J 3기 앵지의 플래너 추천!! 추천 받아서 잘 사용하고 있는 Structured라는 앱이예요! 하루의 태스크를 시간 별로 정리할 수 있어서 좋아요😎🌀

**Reply by yeniful** [reply]
안녕하세요 앵지~ J답게 철두철미한 태스크 정리군요... ! 글 서두에 기수와 닉네임도 함께 알려주시면 감사드리겠습니다-! ㅎㅎ

**Comment by HappyDoran** [comment]
앱 이름: Github

깃허브는 전세계의 여러 사람들이 디자인 토론에 대한 피드백을 공유하거나 몇 줄의 코드를 검토하는 것과 서로에 대한 코드 공유, 토론을 통해 성장을 이뤄 나갈 수 있는 앱입니다.

좋아하는 이유: 깃허브는 개발자들의 성장 가능성을 열어주고, 많은 지식과 정보를 얻어갈 수 있는 장이라고 생각합니다. 제가 생각하는 깃허브의 장점은 본인의 성취도를 얻을 수 있는 commit 확인을 통해 뿌듯함을 얻을 수 있습니다.
![image](https://github.com/user-attachments/assets/1e534877-712d-416c-89fc-d38365a49250)
![image](https://github.com/user-attachments/assets/6156237f-0408-4f6f-b932-d538acd841ce)

**Reply by yeniful** [reply]
ㅋㅋㅋㅋ 개발자의 앱, 깃허브!!!! 저는 협업할 때 바로 알림받을 수 있는게 제일 좋은 것 같아요-! 글 서두에 기수와 닉네임도 함께 알려주시면 감사드리겠습니다-! ㅎㅎ

**Comment by Junu-Park** [comment]
### 3기 `아서`입니다

앱 이름은 뭔가요?: `네이버 지도`

어떤 앱인가요?: `장소 검색` / `네비게이션` / `좋아하는 장소 저장` / `예약&예매` / `다양한 교통 정보 확인` 등 지도 관련 다양한 기능을 한 앱으로 가능!

왜 좋아하나요?: 
- `다양한 기능을 하나의 앱`에서 가능
- `다양한 기능들이 서로 연결`
- `길치.`
- `내가 좋아하는 장소를 보물 창고처럼 저장`해놓고 볼 수 있다!
- `간편한 UI/UX`

![image](https://github.com/user-attachments/assets/fe8fe519-9209-424d-b0b7-eef77df3316a)

**Reply by yeniful** [reply]
네이버 지도 앱 깔끔하고 좋죠 ㅠㅠ!!! 글 서두에 기수와 닉네임도 함께 알려주시면 감사드리겠습니다-! ㅎㅎ

**Comment by cho2-0923** [comment]
이제야 추가하는 기수와 닉네임
3기 레디입니다ㅎㅎ
앱 이름: 무한의 계단
앱 소개: 열심히 계단을 오르는 추억의 게임입니다
좋아하는 이유: 일단 픽셀 디자인을 좋아하구요, 제가 유일하게 잘하는 게임입니다. 운동 안해도 운동한 기분? 저는 하루에 만계단씩 오르고있어요
![IMG_3009](https://github.com/user-attachments/assets/4bb3f12a-f80d-47de-8b42-26d7895341f7)
![IMG_6042](https://github.com/user-attachments/assets/e5b95140-3d3e-4bc4-810e-686db5f6ccf7)

**Reply by yeniful** [reply]
ㅋㅋㅋㅋ 운동 안해도 운동한 기분이라니 !!! 참, 글 서두에 기수와 닉네임도 함께 알려주시면 감사드리겠습니다-! ㅎㅎ

**Comment by Super-BaeNana** [comment]
3기 오전 러너 나나입니다.
![image](https://github.com/user-attachments/assets/a8b207ff-d4b9-4f0b-9371-ac08c72603df)

앱이름: 야놀자
앱소개: 숙소예약
좋아하는 이유: 숙소예약 어플 중 가장 UI가 깔끔하다.

**Reply by yeniful** [reply]
야놀자 인터페이스가 엄청 귀엽고 깔끔하군요 !!!! 외국 서비스들과 대비되는 것 같아요!

**Comment by onegood07** [comment]
![IMG_3946](https://github.com/user-attachments/assets/a057e514-564b-4efd-86ae-16bdbea2724d)
오후반 디케이입니당
앱 이름 : 토스
앱 소개 : 금융권
좋아하는 이유 : 가장 잘 쓰고 있는 앱이고 디자인이 깔끔해서 좋아요

**Reply by yeniful** [reply]
토스!! 너무 유명하고 너무 좋은 앱이죵!!!! 글 서두에 기수와 닉네임도 함께 알려주시면 감사드리겠습니다-! ㅎㅎ

**Comment by someonewho90** [comment]
앱이름: 마디(MADII)
앱소개: 하루의 소확행을 추천해주는 앱
좋아하는 이유: 좋아하는 개발자 이오가 개발하고 소개해준 앱인데 하루에 한 번 알림으로 소소하게 행복을 챙길 수 있는 컨텐츠들을 소개해줍니다.
![IMG_3269](https://github.com/user-attachments/assets/f3f9cc78-89b6-4235-b776-867ce696fa59)
![IMG_3270](https://github.com/user-attachments/assets/4c6def1c-b8ae-484c-adaf-8ddd9d192915)

**Reply by yeniful** [reply]
우와 이오가 개발한 앱이군요!!! 글 서두에 기수와 닉네임도 함께 알려주시면 감사드리겠습니다-! ㅎㅎ

**Reply by anjiniii** [reply]
어머! 어머!! 여기서 마디를 만나다니!! 너무너무 감사합니다 🍀🍀

**Comment by sunhaleeRiley** [comment]
![IMG_3235](https://github.com/user-attachments/assets/71b87c81-ef3a-4827-b2ac-3f85b0bdedc7)

모두가 잘쓰시겠지만 .. 전 진짜 투두메이트 최근에 입문하고 아카데미 러너들과 일정을 공유하며 너무 잘쓰고 있습니다..!!!

**Reply by yeniful** [reply]
투두메이트.. 투두 앱 지박령이지만 저도 너무 좋아하는 앱이에요! 글 서두에 기수와 닉네임도 함께 알려주시면 감사드리겠습니다-! ㅎㅎ

**Comment by seeeRini** [comment]
![IMG_8585](https://github.com/user-attachments/assets/5a0f0393-2b7b-4482-bf87-0eee7463ed7e)
![IMG_8584](https://github.com/user-attachments/assets/8fe47ee4-b283-4396-9c81-e0b35063bade)
안녕하세여! 오전반 리니입니다 👾
아카데미 러너라면 모를수가 없는 체킹앱!
저는 아침마다 아슬아슬하게 사용하고 있는데요🤔
그럼에도 지각은 아직 한번도 없다는 사실!
그래서 사용할수록 더 뿌듯해지는 것 같습니다 :)

여러분들 체킹 잊지마세요 👻👻👻

**Reply by yeniful** [reply]
ㅋㅋㅋㅋㅋ 체킹이 등장하다니!!! 여러분들 체킹 잊지마세요 👻👻👻

**Comment by leeymin130** [comment]
3기 오후반 마리입니다! 제가 좋아하는 앱은 바로..듀오링고입니다^^ 
![IMG_9543](https://github.com/user-attachments/assets/839ee9ee-b470-4ba7-a31d-91da9c0a73e4)

노력 없이 언어공부 하고싶다? 답은 듀오링고입니다. 다들 듀오링고하세요~
![IMG_9542](https://github.com/user-attachments/assets/40368eca-506f-4b0c-9b8d-ece42497430e)

위젯도 너무 귀엽습니다.
![IMG_9541](https://github.com/user-attachments/assets/983dc5f0-e059-48f8-b1af-550b294a9269)

**Comment by Younkyum** [comment]
3기 오전 온브입니다. 
삶의 행복을 가져다주는 Weverse 추천합니다. 
![IMG_3444](https://github.com/user-attachments/assets/d2d1b0cc-e7ee-4bc7-bb1a-01d6e308533b)

**Comment by Seon-U** [comment]
![image](https://github.com/user-attachments/assets/fd21865f-057b-4f78-ae7b-0d4275d473c7)


3기 나우입니다!
혹시 이앱을 아시나요? 
Fabula라는 이 앱은 swiftUI 기반의 다양하고 신기한 코드들을 모아 구현모습과 코드를 동시에 모여주는 어플입니다. 
![image](https://github.com/user-attachments/assets/8d25ba99-c925-4b0c-956a-a848c17d2613)
![image](https://github.com/user-attachments/assets/a94d4b31-e3ba-401a-9b01-b1a74b0cfacf)
특히 애니메이션 자료가 많아서 참고하거나 심심할 때 들어가 보기 좋아요! 깃허브 기반으로 누구나 코드를 추가할 수 있기 때문에 기발한 코드를 작성하셨다면 참여해보셔도 좋을듯합니다!

**Comment by tenedict** [comment]
![image](https://github.com/user-attachments/assets/2df2703f-de56-4ca7-9421-07bbfd74f4fc)

언제봐도 재미있는 유튜브~~

**Comment by JinBella0** [comment]
![image](https://github.com/user-attachments/assets/745461eb-f686-469c-a141-12dc0366968c)

3기 오전반 벨라 입니다
앱 이름 : 뱅크샐러드
앱 소개 : 가계부
좋아하는 이유 : 가장 유심있게 보는 앱! 한 달 지출 관리와 지출비 분석을 해줘서 유용해요

**Comment by norang00** [comment]
3기 노랑입니다
앱이름: 삐삐 Bbibbi
앱소개: 가족들에게 생존신고 할 수 있는 어플이에요! 친구들과는 여러 SNS로 소통하는데 은근 부모님과 소통이 안된다고 느낀적이 많아요. 그런데 이 앱을 쓰면 더 재밌게 소통할 수 있어요!
![image](https://github.com/user-attachments/assets/6f5feb18-0bed-4401-aaf7-7d51426914d8)

**Comment by yoondj98** [comment]
3기 오전반 제라스입니다! 👋🏻 🤖 👋🏻
KBO 직관러들을 위한 최고의 꿀 앱 '자리어때'를 소개합니다~!
**각 구장마다의 좌석마다의 시야를 제공**해주어 좌석을 선택할 때 큰 도움이 되는 소중한 앱입니다.

<img width= "250" src="https://github.com/user-attachments/assets/300ac22a-1887-4541-a9a1-31f91c81b98d"/>
<img width= "250" src="https://github.com/user-attachments/assets/11b7f7d3-0e05-4889-9e21-66e740457134"/>
<img width= "250" src="https://github.com/user-attachments/assets/b0142768-ba1c-422e-944a-78b208c85dee"/>

**Comment by zaehorang** [comment]
3기 오후 호랑입니다.
앱 이름: 민트 주간
 어떤 앱인지: iOS 개발자를 위한 앱입니다.
<img src="https://github.com/user-attachments/assets/d52689c2-46d1-48c3-879f-5c114451b0e0" width="200">

왜 좋아하는지:
iOS 개발 관련 아티클을 한 번에 보기 좋아요.

**Comment by hyeonheebee** [comment]
3기 오후 웰디입니다. 
앱 이름: Numvert 
![스크린샷, 2024-08-13 오후 2 52 47](https://github.com/user-attachments/assets/d772c527-fb31-49be-9481-cc5c908a1457)
![스크린샷, 2024-08-13 오후 2 53 39](https://github.com/user-attachments/assets/014a28c9-c4a9-46b7-860e-56b99b0e7e4d)

어떤 앱인지: iOS 앱 
왜 좋아하는지 : 아주 직관적이고 깔끔한 UI로 외국인들, 혹은 한국인들도 큰 숫자 읽기를 가능하게 합니다. 그리고 해당 앱을 만든 팀이 귀엽습니다 >_<

**Comment by kyu0122** [comment]
3기 오후 루카스입니다.
앱이름: Duolingo
좋아하는 이유: 1년 넘게 사용하면서 재미있게 외국어를 공부할 수 있어서 너무 좋았어요! 특히 단순한 단어 정리나 reading이 아니라 speaking, listening, writing까지 모두 학습할 수 있어서 자투리시간을 알차게 활용할 수 있어서 너무 좋았어요.

![IMG_1995](https://github.com/user-attachments/assets/c335410f-a2a9-4506-8e91-5e5893085926)
![IMG_1994](https://github.com/user-attachments/assets/9dd100ad-e22e-4a29-906f-876341f052fc)

**Comment by dbqls200** [comment]
<img src="https://github.com/user-attachments/assets/a4a41714-a89d-40fa-b146-fc6b5ab43a4b" width="200">
  
오후 3기 **유디**입니다 람쥐
- 앱 이름 : 간단
- 간헐적 단식 시간을 기록할 수 있는 앱입니당 😎
- 매일 시간 체크하느라 귀찮았는데 기록 도와줘서 삶의 질 향상됐으요 💪🏻

**Comment by saejinpark** [comment]
3기 제패토입니다
앱이름: 애플뮤직
앱소개: 애플에 기본 음악앱입니다
좋아하는 이유: 선망하는 앱이에요 ㅎㅎ..

![image](https://github.com/user-attachments/assets/f761551f-ce02-4efc-b20a-ac4a2a6da77a)

**Comment by daminoworld** [comment]
![IMG_1044](https://github.com/user-attachments/assets/a4465be9-c1d2-4f43-8fb4-55e5a3d27d0b)

안녕하세요 3기 비전입니다
앱이름: Remote+
- 아이폰, 아이패드로 맥을 원격 조종할 수 있는 앱입니다.
- 요즘에는 못쓰고 있지만 집에서 맥으로 넷플릭스 틀어 놓고 누워서 휴대폰으로 영상 조정 할때 편해요..ㅎㅎ

**Comment by morningfest** [comment]
![B071DBBD-83A2-4456-B1A4-461F55C5CF7D_1_201_a](https://github.com/user-attachments/assets/97c373fc-207d-4ce3-849b-2198e3cc268d)
![43F83B4E-9CD2-495F-80D3-468B7CA9EE21_4_5005_c](https://github.com/user-attachments/assets/2122152e-3ad2-466f-8714-643db52b3e7d)
3기 김리입니다.
이거 덕분에 저희 팀 캐릭터를 만들었어용 추천추추천

**Comment by Lyla-Dev** [comment]
3기 민입니다!
![IMG_2262](https://github.com/user-attachments/assets/12b621d2-264f-4367-9ab2-b2f59fe28327)
![IMG_2261](https://github.com/user-attachments/assets/237e756f-f186-4133-ac4f-3589ce289754)

삐삐라는 앱인데, 가족과 생존신고용으로 사용하고 있습니다! 가볍게 안부 확인할 수 있어서 좋아요

**Comment by ye-gang-jjang** [comment]
<img width="1205" alt="image" src="https://github.com/user-attachments/assets/54b461b7-bd17-466d-acc4-4cba0b570932">

3기 라프입니다

쇼핑할 때 사용하는 앱입니다
